Don't obfuscate email addresses

gfontenot opened the pull request:
This really doesn't matter too much, and leaving the obfuscation in
means that we get warnings in RSS feeds if we're using a mail link. I
think it makes the most sense to turn this off.

Learned in this PR:
- `hiding`. Very cool.
- That syntax for creating a new instance of something with a changed
  value. I have no idea what that is or why it works and that scares me
  a little but hey, here we are.

gfontenot committed:
Don't obfuscate email addresses

This really doesn't matter too much, and leaving the obfuscation in
means that we get warnings in RSS feeds if we're using a mail link. I
think it makes the most sense to turn this off.

Learned in this PR:

 - `hiding`. Very cool.
 - That syntax for creating a new instance of something with a changed
   value. I have no idea what that is or why it works and that scares me
   a little but hey, here we are.

sidraval commented:
:+1:

bernerdschaefer commented on site.hs:
```diff
@@ -1,6 +1,7 @@
 {-# LANGUAGE OverloadedStrings #-}
 
-import Hakyll
+import Hakyll hiding (pandocCompiler)
```
This is horrible, but seems useful.

gfontenot replied:
Hahaha. Yes. I felt bad about this, not going to lie.

gfontenot committed:
Use explicit imports for Pandoc

We don't need _all_ of Pandoc in here mucking up the place. Lets just
import the two things we need.
